modules/cudacodec/src/frame_queue.cpp

```diff
@@ -55,18 +55,26 @@ cv::cudacodec::detail::FrameQueue::~FrameQueue() {
         delete[] isFrameInUse_;
 }
 
-void cv::cudacodec::detail::FrameQueue::init(const int _maxSz) {
+void cv::cudacodec::detail::FrameQueue::init(const int _maxSz, const bool force) {
     AutoLock autoLock(mtx_);
+    if (isFrameInUse_) {
+        if (force)
+            delete[] isFrameInUse_;
```

Comment:
Am I right that it cases memory leak, if `isFrameInUse_` defines alive frames?

Reply:
Yes even though it is an internal class which is only called once it was vunerable to a memory leak if init() is called more than once.

Reply:
Correct me, if I miss something, but the path and the module itself does not use `init` with `force=true`. I propose to remove it from `init` as it's dead code and dangerous flag.

Reply:
No you are correct.  Force is not used and can be removed because it could be dangerous to have the facility to initialize a frameque object twice.  It may be better if required in the future to have a specific reinitialization routine which performs safety checks, if the size of the que needs to be altered mid-decode.